src/emails/components/EmailDataPointCount.tsx

```diff
@@ -0,0 +1,126 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* c8 ignore start */
+
+import React from "react";
+import { ExtendedReactLocalization } from "../../app/functions/l10n";
+import { DashboardSummary } from "../../app/functions/server/dashboard";
+import { hasPremium } from "../../app/functions/universal/user";
+import { SubscriberRow } from "knex/types/tables";
+
+type Props = {
+  l10n: ExtendedReactLocalization;
+  utmCampaignId: string;
+  utmContentSuffix: string;
+  dataSummary: DashboardSummary;
+  subscriber: SubscriberRow;
+};
+
+export const DataPointCount = (props: Props) => {
+  const unresolvedDataBreaches = props.dataSummary.dataBreachUnresolvedNum;
+  const unresolvedDataBreachesAndBrokers =
+    props.dataSummary.dataBreachUnresolvedNum +
+    props.dataSummary.dataBrokerManuallyResolvedNum;
```

Comment:
Still, is that number equal to the unresolved data point count, as per [the previous calculation](https://github.com/mozilla/blurts-server/blob/9305737fc5e66a8813da3a85a78af370bf736145/src/emails/templates/breachAlert/BreachAlertEmail.tsx#L355-L361)?

(Yes, that code is gnarly - I'd really like to redo the DashboardSummary data structure to avoid this, but it is what it is now...)

Reply:
It is; it's even more convoluted than what I linked to above, but the `reduce` is [here](https://github.com/mozilla/blurts-server/blob/9305737fc5e66a8813da3a85a78af370bf736145/src/app/components/client/Chart.tsx#L58-L61), and the data it's reducing over [here](https://github.com/mozilla/blurts-server/blob/9305737fc5e66a8813da3a85a78af370bf736145/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/DashboardTopBanner/index.tsx#L38-L47).